src/components/app/details/appDetails/SourceInfo.tsx

```diff
@@ -96,11 +96,11 @@ export function SourceInfo({
                     </Tippy>
                 )}
                 {appDetails?.deploymentAppDeleteRequest && (
-                    <>
+                    <div data-testid="deleteing-argocd-pipeline">
```

Comment:
Also, check & confirm that there's no UI impact of adding this div

Reply:
No impact